homeassistant/components/blink/binary_sensor.py

```diff
@@ -21,12 +22,14 @@
     TYPE_CAMERA_ARMED,
     TYPE_MOTION_DETECTED,
 )
+from .coordinator import BlinkUpdateCoordinator
 
 _LOGGER = logging.getLogger(__name__)
 
 BINARY_SENSORS_TYPES: tuple[BinarySensorEntityDescription, ...] = (
     BinarySensorEntityDescription(
         key=TYPE_BATTERY,
+        translation_key="battery",
```

Comment:
These changes aren't needed and should not be a part of this PR tbh

Reply:
They kind of are... otherwise we generate useless _2, _3 names.  It didn't do that before (at least before moving to the AIO version of the API).  I suppose I can remove it from here and create a new PR with just this?

Reply:
More like, as far as I see the naming in the old integration should work perfectly. If you're generating duplicates, there might be an unique id issue